app/config/config.yml

```diff
@@ -199,3 +199,42 @@ scheb_two_factor:
 kphoen_rulerz:
     executors:
         doctrine: true
+
+fos_elastica:
+    clients:
+        default: { host: localhost, port: 9200 }
+    serializer:
+        callback_class: FOS\ElasticaBundle\Serializer\Callback
+        serializer: serializer
+    indexes:
+        wallabag:
+            client: default
+            settings:
+                index:
+                    analysis:
+                        analyzer:
+                            custom_french_analyzer:
+                                type: custom
+                                tokenizer: letter
+                                filter: ["asciifolding", "lowercase", "french_stem", "stop_fr"]
```

Comment:
The `french_stem` token filter does not exist, you don't have any stemming on this analyzer.

I suggest you use elision too, instead of `stop_fr`. Have a look at [this example](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-lang-analyzer.html#french-analyzer).

Also, why tokenizing for french only? I'm not sure the contents are mono-lang?